tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -136,8 +137,11 @@ sub do_networking_tests {
 
     # Check if guest is booted and ssh service is started, not needed for sle-micro
     if (check_var('DISTRI', 'sle')) {
-        die "SSH is not reacheble" if (script_retry("nmap $vm_ip -PN -p ssh | grep open", delay => 10, retry => 12, timeout => 360) != 0);
+        # If nmap is not installed, install it
+        install_package("nmap", trup_reboot => 1) if (script_run("command -v nmap"));
```

Comment:
This will require pkg installation and reboot. Seems a better location for it would be in earlier system preparation step, eg [system_prepare](https://openqa.suse.de/tests/17975843/modules/system_prepare/steps/1/src).

Reply:
I tried to run with SCC_ADDONS=sdk,legacy, but it didn't install nmap command in system_prepare test module. 
https://openqa.suse.de/tests/18050075

Reply:
I just thought that installing the pkg in system_prepare would share reboot there, so more efficient, and it is the file for preparing system(there are other pkg installation there).

> I tried to run with SCC_ADDONS=sdk,legacy, but it didn't install nmap command in system_prepare test module.
https://openqa.suse.de/tests/18050075

BTW, running with SCC_ADDONS=sdk,legacy is not equivalent to what I suggested. I meant to MOVE the code in this PR from one file to another, not skipping the nmap installation.

Anyway, it is not big deal. I share comment, but you can decide after comparison.

Reply:
Yes, I still think that checking the nmap package before running command is a better approach. And it doesn't need to reboot after installing the package as it's in the condition "DISTRI=sle".